### ISSUE TITLE
DNS not working anymore on Fedora 40/Podman 5.0

### ISSUE THREAD

**Issue by francoism90** [containers/podman-compose#904]
**Describe the bug**
DNS is not working at all anymore after upgrading to the latest Fedora 40 beta release.

**To Reproduce**
1. Create network bridge
2. Set this network in docker-compose.yml
3. podman compose up -d

**Expected behavior**
It does attach some DNS, but it doesn't work at all.
It just doesn't seem to route anything.

**Actual behavior**
DNS working

**Output**

```
$ podman-compose version
using podman version: 5.0.0
podman-compose version  1.0.6

$ podman-compose up
...

```

**Environment:**
 - OS: Linux / WSL / Mac
 - podman version: 
 - podman compose version: (git hex)

**Additional context**

It's weird, because it was working fine a few days ago. I did remove the netwerk, enabled IPv6, hotspot, .. it just doesn't connect to anything.

**Comment by benyaminl** [comment]
Have you check the network backend using aadvark? and the network generated is supporting dns using podman network inspect?

**Comment by francoism90** [comment]
@benyaminl This is actually a Podman bug with the new network plugin pasta.

It will hopefully be solved with Podman 5.0.1.

Thanks for looking into this, I'll close this issue. :)
